### PR TITLE
Update Navie chat interface

### PR DESCRIPTION
**Pull request by ahtrotta**
- Updates Navie styling (see [here](https://github.com/getappmap/appmap-js/pull/1589))
- Handles event from clicking `Looking for code visualizations?` by opening the AppMaps tree (see picture).

image

### CODE REVIEW

File: src/tree/index.ts [hunk 1]
```diff
@@ -70,6 +70,16 @@ export default function registerTrees(
...
 
+  context.subscriptions.push(
+    vscode.commands.registerCommand('appmap.view.focusAppMapTree', () => {
+      localAppMapsTree.reveal(undefined as unknown as AppMapTreeItem, {
+        select: true,
+        focus: true,
+        expand: true,
+      });
+    })
+  );
```
Comment (dustinbyrne) [inline src/tree/index.ts:81]: There should be a command created by VSCode automatically for the view, e.g. `viewName.focus`
```
appmap.views.appmaps.focus
```

File: src/webviews/chatSearchWebview.ts [hunk 2]
```diff
@@ -100,6 +100,9 @@ export default class ChatSearchWebview {
         case 'open-record-instructions':
           await vscode.commands.executeCommand('appmap.openInstallGuide', RecordAppMaps);
           break;
+        case 'show-appmap-tree':
+          await vscode.commands.executeCommand('appmap.view.focusAppMapTree');
```
Comment (dustinbyrne) [inline src/webviews/chatSearchWebview.ts:104]: Use `appmap.views.appmaps.focus`

File: src/tree/index.ts (hunk 1 above)
Comment (ahtrotta) [reply src/tree/index.ts]: Cool, I didn't know about that, thanks :+1:

File: src/webviews/chatSearchWebview.ts (hunk 2 above)
Comment (ahtrotta) [reply src/webviews/chatSearchWebview.ts]: :+1:

**Pull request merged**